common/djangoapps/entitlements/api/v1/views.py

```diff
@@ -275,6 +275,11 @@ def partial_update(self, request, *args, **kwargs):
             )
         support_details = request.data.pop('support_details', [])
 
+        # If a patch request does not explicitly update an entitlement's refundability status, we want to ensure that
+        # changes made to other attributes of the entitlement do not implicitly change its ability to be refunded.
+        if request.data.get('refund_locked', None) is None:
```

Comment:
2 things here, I believe get defaults to returning None and also you can probably just do

`if not request.data.get('refund_locked')`

Reply:
Does get not raise an error if you don't give it a default return value?

Also I want to explicitly check for None, because the value could also be False, which I would want to use when updating the entitlement

Reply:
Just confirmed that it does default to None, and does not raise an error.